lib/Parser.js

```diff
@@ -299,6 +299,11 @@ function parseHeader(message, data, headerStart, headerEnd)
         message.event = parsed;
       }
       break;
+    // allow multiple call info headers
```

Comment:
Please, keep the comments style (uppercase the first letter and end with dot).

Reply:
Will be changed